docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -143,6 +143,75 @@ game.properties:        158 bytes
 ui.properties:          83 bytes
 ```
 
+You can pass in the `--from-env-file` to create a ConfigMap from an env-file (i.e. a Docker .env file), for example:
+```shell
+# Env-files contain a list of environment variables.
+# These syntax rules apply:
+#   Each line in an env file has to be in VAR=VAL format.
+#   Lines beginning with # (i.e. comments) are ignored.
+#   Blank lines are ignored.
+#   There is no special handling of quotation marks (i.e. they will be part of the ConfigMap value)).
+
+
+cat docs/tasks/configure-pod-container/game-env-file.properties
```

Comment:
rather than keeping this content in two places, you could {% include game-env-file.properties %} to reference the content you have in a separate file in this PR. 

I personally prefer to include the content directly into the tutorial and not have separate files, but I'll the choice is yours if you think there's further value to having that file separate.

Reply:
Thanks yeah, I'd say adding the file gives a closer example to a real scenario with the env files.